lesson16/js/main.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+const categoryTab = document.getElementById('js-tab');
+const neswArticle = [];
+
+function showLoadingImg() {
+	const newsContentArea = document.querySelector('.news-area');
+	const createLoadingBox = document.createElement('div');
+	createLoadingBox.classList.add('ly_loading');
+	createLoadingBox.id = 'js-loading';
+	const loadingImg = document.createElement('img');
+	loadingImg.src = 'images/loading-circle.gif';
+	loadingImg.classList.add('el_loadingImg');
+	newsContentArea.appendChild(createLoadingBox).appendChild(loadingImg);
+}
+
+function hideLoadingImg() {
+	const loading = document.getElementById('js-loading');
+	loading.remove();
+}
```

Comment:
IMO
If something is used only once, it does not have to be declared as a constant. This is because it is a waste of memory.
```suggestion
function hideLoadingImg() {
	document.getElementById('js-loading').remove();
}
```

Reply:
Thank you!
As you advised. I cleaned up the code by eliminating superfluous variable declarations.

[4b052e9](https://github.com/nyattsu72/hanson-frontend-cource/pull/21/commits/4b052e98a8ecfa42de5ec2250b7c8593165831ac)